### ISSUE TITLE
NO power display

### ISSUE THREAD

**Issue by RuiYangQuan** [amanusk/s-tui#179]
### Step 1: Describe your environment

  * System (Intel/AMD/Raspberry Pi etc): Raspberry Pi 
  * OS version: ubuntu18
  * s-tui version: 2.0
  * Installation method(pip/PPA/source code): pip
  
#### Observed Results:

  * What happened? (Traceback of a crash/Image/description):
  
I installed an S-TUi on the Raspberry PI, but no power display
```
* Replace this with debug output *
```
  
#### Steps to reproduce:
1. _____
2. _____
3. _____

```
* Replace this with relevant code to reproduce the problem *
```

**Comment by amanusk** [comment]
Powe displays is only available on Intel and some Ryzen cpus.

**Comment by sq-pyxisai** [comment]
I didn't see power until I ran s-tui as root.

**Comment by amanusk** [comment]
Correct, power works with Intel and some Ryzen CPUs but tequires root to fetch the data.

**Comment by KylinC** [comment]
sudo is all you need!
